ledger/participant-integration-api/src/main/scala/platform/DispatcherState.scala

```diff
@@ -24,53 +24,54 @@ class DispatcherState(dispatcherShutdownTimeout: Duration)(implicit
 ) {
   private val logger = ContextualizedLogger.get(getClass)
   @volatile private var dispatcherStateRef: DispatcherState.State =
-    DispatcherNotInitialized
+    DispatcherNotRunning
 
-  def initialized: Boolean = dispatcherStateRef match {
-    case _: DispatcherState.DispatcherOperational => true
-    case DispatcherState.DispatcherNotInitialized | DispatcherState.DispatcherShutdown => false
+  def getDispatcher: Dispatcher[Offset] = dispatcherStateRef match {
+    case DispatcherNotRunning => throw dispatcherNotRunning()
```

Comment:
this is possible to happen during normal operation. will this exception bubble up as a serlf-service error code?

Reply:
Yes, but I think as INTERNAL atm. Good observation. I will convert to a typed self-service error code.

Reply:
Done.